resources/models/unit.py

```diff
@@ -33,6 +33,22 @@ def managed_by(self, user):
 
         return self.filter(via_unit_group | via_unit).distinct()
 
+    def administered_by(self, user):
+        if not is_authenticated_user(user):
+            return self.none()
+
+        if is_general_admin(user):
+            return self
+
+        unit_group_q = Q(
+            unit_groups__authorizations__in=(
+                user.unit_group_authorizations.admin_level()))
+        unit_q = Q(
+            authorizations__in=(
+                user.unit_authorizations.admin_level()))
+
+        return self.filter(unit_group_q | unit_q).distinct()
```

Comment:
This new method should not be needed, since managed_by already considers the admins as users that can manage the units.

Reply:
@suutari-ai Good cathes, the specification changed at some point, which apparently caused some confusion. However, I have changed the code now.